source/sass/components/article-page.scss

```diff
@@ -23,18 +23,12 @@
     max-width: 100%;
     width: 100%;
   }
+}
 
-  .article-links {
-    display: flex;
-    flex-direction: column;
-    &__two-col {
-      @media (min-width: $bp-md) {
-        display: grid;
-        grid-template-rows: auto auto auto auto;
-        grid-auto-flow: column;
-        grid-gap: 10px;
-      }
-    }
+.article-links-two-column {
```

Comment:
if there are more than 5 items, the article links  div is then given the "Article-links-two-column" class which gives it a column count of two, only above the medium breakpoint. 

If it is tablet-sized or under, the links display block one above another.